src/services/SignatureRequestProcessor.ts

```diff
@@ -8,7 +8,7 @@ import { SafeApiQueue } from "./SafeApiQueue.js";
 type SignatureRequest =
   Database["public"]["Tables"]["signature_requests"]["Row"];
 
-export class SignatureRequestProcessor {
+export default class SignatureRequestProcessor {
```

Comment:
Pet epeve, but in general I like names exports if the file also contains types, because we might need to use that type as well. In this case, the type is a direct import of another type.

So either:
a) the type declaration is redundant
- the type needs some syntactic sugar and is usefull in other places as well

In the case of a) this is at least more readable, in the case of b) export the type and use named exports.

Up to you, there was an explicit commit on this hence the feedback